roles/patch_host_config/templates/patch-worker-partition.j2

```diff
@@ -0,0 +1,28 @@
+#jinja2:trim_blocks: True, lstrip_blocks: True
+{
+  "ignition": {
+    "version": "3.2.0" 
+  },
+  "storage": {
+    "disks": [
+      {% for disk in disks %}
+      {
+        "device": "{{ disk.device }}",
+        "partitions": [ 
+          {% set partitions = ((disk.extra_partitions | default({})).items() | list) %}
+          {% if disk.device == (installation_disk_path | default(host.json.installation_path)) %}
```

Comment:
@hhamid93 Did you check the json returned from the host endpoint to check this is the correct path?